.github/workflows/formatting-checks.yml

```diff
@@ -0,0 +1,43 @@
+name: Formatting Checks
```

Comment:
Should we name it, even if verbose, "Backend & Scripts Formatting Checks" to make obvious it is not meant to format frontend?

Reply:
Sure!  Done (well, a slightly shorter version! :slightly_smiling_face:  )